src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -104,14 +104,18 @@ private static async Task InitializeInstance(ITestOutputHelper output)
         var chromeDriverPathEnvVar = Environment.GetEnvironmentVariable("CHROMEWEBDRIVER");
         if (!string.IsNullOrEmpty(chromeDriverPathEnvVar))
         {
-            chromeDriverArg = $"--javaArgs=-Dwebdriver.chrome.driver={chromeDriverPathEnvVar}/chromedriver";
+            var configContent = File.ReadAllText(seleniumConfigPath);
```

Comment:
FYI, this is working around what is likely a bug in selenium-standalone introduced by https://github.com/webdriverio/selenium-standalone/commit/d13777c13c153e26d57563ae8aca0f998c0e01b2#diff-20cb449399750b160a4adfac13aa25ed97c9faacf30bee11ed4a16870f3ff4bfR103
It expands the individual characters of the command line argument for `--javaArgs`, so `-Dblah` becomes `['-','D','b','l','a','h']` which then fails since each value in the array is passed as an argument to java and `-` isn't a valid argument.

Might be worth filing an issue.